src/test/java/software/amazon/smithy/lsp/TestUtils.java

```diff
@@ -0,0 +1,59 @@
+package software.amazon.smithy.lsp;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import junit.framework.AssertionFailedError;
+import org.eclipse.lsp4j.Location;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+import software.amazon.smithy.lsp.ext.SmithyProjectTest;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
Not necessary, but FYI you can actually make custom JUnit assertions. See for example https://github.com/smithy-lang/smithy/blob/main/smithy-model/src/test/java/software/amazon/smithy/model/ShapeMatcher.java

Reply:
I'd like to move the language server to hamcrest and junit5 like the Smithy repo, maybe this can be a part of that. Seems useful